nucleus/sensorfusion_utils.py

```diff
@@ -0,0 +1,19 @@
+import open3d as o3d
```

Comment:
This needs to be added as a project dependency using 

`poetry add`

I think it's as simple as

`poetry add open3d`